hub/cli/auth.py

```diff
@@ -0,0 +1,56 @@
+from hub.client.utils import write_token, remove_token
+from hub.client.client import HubBackendClient
+from hub.util.exceptions import AuthenticationException
+import click
+
+
+@click.command()
+@click.option("--username", "-u", default=None, help="Your Activeloop username")
+@click.option("--password", "-p", default=None, help="Your Activeloop password")
+def login(username: str, password: str):
+    """Log in to Activeloop"""
+    click.echo("Log in using Activeloop credentials.")
```

Comment:
I really don't see a point in taking all strings out of code. I'm not a fan of creating single-use string constants.

HUB_LOGIN_PROMPT = "Log in using Activeloop credentials."
HUB_PASSWORD_HELP = "Your Activeloop password"

Things like this will never get reused.

Reply:
It's not about re-use but the ability to change the strings later in the future. 
I wanted to have this reviewed by others as this helps us in making the entire code more useful and easy to understand errors.

Also, we can control typos and a lot of other benefits as well.

Reply:
Again, I'd like to say that having a separate file with all of these out of context strings won't make sense for marketing as well. They'll have to look up where the string actually occurs before making any changes. This will lead them to the original file and defeats the purpose of having the string constants for these anyway.